src/videojs-http-streaming.js

```diff
@@ -168,11 +175,11 @@ const setupEmeOptions = (hlsHandler) => {
   const player = videojs.players[hlsHandler.tech_.options_.playerId];
 
   if (player.eme) {
-    player.eme.options = videojs.mergeOptions(player.eme.options, emeOptions(
+    player.eme.options = emeOptions(
```

Comment:
Removed this because of the following scenario: given a playlist with two DRM sources, the first has in-manifest DRM info and the second does not, the first source's data would carry over into the second and prevent playback.

Reply:
In the long term we'll look into making contrib-eme a source handler

Reply:
I know we discussed the possibility of removing player.eme.options entirely, and I think we should go through with that on a major version of videojs-contrib-eme. However, I think we should be careful about completely replacing player.eme.options right now. With this behavior if someone had used the top level eme plugin options, they will all be overwritten. I think instead we should find a way to pass along the new options from VHS as part of the source. Maybe via modifying the source object itself.

Reply:
The source options merge with the plugin options in contrib-eme so it would behave the same way it was before where we merge them here

Reply:
Instead I'll default `pssh: ''` and do the merge